### PR TITLE
bugfix/duplicate-columns

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** Issue #119 

**This PR will result in the following new package version:** `v0.12.0`

This will technically be a breaking change since it will remove (via a coalesce) existing impacted fields if duplicates are identified. As such, I would feel more comfortable with this being a breaking change so customers are aware of the upgrade being applied.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

- This release includes breaking changes as a result of upstream changes within the `v0.12.0` release of the dbt_hubspot_source package. Please see below for the relevant breaking change release notes from the source package. ([PR #120](https://github.com/fivetran/dbt_hubspot/pull/120))
- The following models in the dbt_hubspot_source package now use a custom macro to remove the property_hs_ prefix in staging columns, while also preventing duplicates. If de-prefixed columns match existing ones (e.g., `property_hs_meeting_outcome` vs. `meeting_outcome`), the macro favors the `property_hs_`field, aligning with the latest HubSpot API update. ([PR #115](https://github.com/fivetran/dbt_hubspot_source/pull/115))
  - `stg_hubspot__engagement_call`
  - `stg_hubspot__engagement_company`
  - `stg_hubspot__engagement_contact`
  - `stg_hubspot__engagement_deal`
  - `stg_hubspot__engagement_email`
  - `stg_hubspot__engagement_meeting`
  - `stg_hubspot__engagement_note`
  - `stg_hubspot__engagement_task`
  - `stg_hubspot__ticket`
  - `stg_hubspot__ticket_company`
  - `stg_hubspot__ticket_contact`
  - `stg_hubspot__ticket_deal`
  - `stg_hubspot__ticket_engagement`
  - `stg_hubspot__ticket_property_history`
  
### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [X] dbt test
- [n/a] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

These steps were validated by recreating the issue with the seed data for our integration tests and also via validation with the customer that the fix does in fact resolve the error they are seeing.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [n/a] DECISIONLOG updates have been updated (if applicable)
- [n/a] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
2️⃣

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 packages:
-  - package: fivetran/hubspot_source
-    version: [">=0.11.0", "<0.12.0"]
+  # - package: fivetran/hubspot_source
+  #   version: [">=0.11.0", "<0.12.0"]
+
+  - git: https://github.com/fivetran/dbt_hubspot_source.git
+    revision: bugfix/duplicate-columns
+    warn-unpinned: false
```
Comment (fivetran-catfritz) [inline packages.yml:7]: Standard reminder to update before mege

File: packages.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 packages:
-  - package: fivetran/hubspot_source
-    version: [">=0.11.0", "<0.12.0"]
+  # - package: fivetran/hubspot_source
+  #   version: [">=0.11.0", "<0.12.0"]
```
Comment (fivetran-catfritz) [inline packages.yml:3]:
```suggestion
  #   version: [">=0.120", "<0.13.0"]
```

**Review by fivetran-catfritz — APPROVED** [review]
lgtm!

**Comment by blackary** [comment]
When can we expect this release to be available on dbt Package Hub? Does that happen automatically with some delay? Currently the latest version available [there](https://hub.getdbt.com/fivetran/hubspot/latest/) is 0.11.0

**Comment by fivetran-joemarkiewicz** [comment]
Hi @blackary thanks for reaching out. There were some issues with the dbt hub yesterday that delayed this update from being rolled out. I got word from dbt Labs that the hub is back up and working! I have since deployed this new version (`0.12.0`) and it should be live on the dbt hub at the top of the hour.

**Pull request merged**